code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -1,3 +1,5 @@
+#define MAX_LIMB_DAMAGE 200
```

Comment:
Using this define you've given all non-head limbs 200 max damage.

That means that a player can outright die from damage to a single limb and very weak explosions can now instantly kill players as limbs contribute more than 50 damage each.

These changes need reverting or fixing to maintain parity to current code.

Reply:
I've got a different plan in mind for limbs, don't worry.